module/item/item.mjs

```diff
@@ -1210,7 +1211,9 @@ export class HeroSystem6eItem extends Item {
             }
 
             if (dc > 0) {
-                if (item.system.killing) {
+                if (item.system.XMLID === "TRANSFORM") {
+                    item.system.damage += "T";
```

Comment:
Do we really want to have a separate type for this arguably because it doesn't do a different type of damage. You might want to see the ugly way adjustment powers are done with an active effect and do things that way.

Reply:
At least in 6e, Transform is specified as applying 'Transform "damage"' (with the word "damage" actually in quotes). I'm comfortable with whatever direction we want to this though.

Reply:
Fixed